containers/src/Kokkos_DualView.hpp

```diff
@@ -865,4 +865,27 @@ void deep_copy(
 
 }  // namespace Kokkos
 
+//----------------------------------------------------------------------------
+//----------------------------------------------------------------------------
+
+namespace Kokkos {
+
+//
+// Non-member resize and realloc
+//
+
+template <class... Properties, class... Args>
+void resize(DualView<Properties...>& dv, Args&&... args) noexcept(
+    noexcept(dv.resize(std::forward<Args>(args)...))) {
+  std::move(dv).resize(std::forward<Args>(args)...);
```

Comment:
Why do you cast to a rvalue reference?

Reply:
uhhh... you're right, I shouldn't. I think it's because it wasn't a reference at one point, and so I was moving out of the parameter storage, but I think then I realized it needed to be a reference anyway. Good catch

Reply:
```suggestion
  dv.resize(std::forward<Args>(args)...);
```